### PR TITLE
Event Log Tests disabled for Windows 7

### PR DESCRIPTION
**Pull request by Anipik**
Relates to #24837 and #24690.

### CODE REVIEW

File: src/System.Diagnostics.EventLog/tests/Helpers.cs [hunk 1]
```diff
@@ -7,6 +7,19 @@ namespace System.Diagnostics.Tests
     internal class Helpers
     {
         public static bool IsElevatedAndSupportsEventLogs { get => AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
-        public static bool SupportsEventLogs { get => PlatformDetection.IsNotWindowsNanoServer; }
+        public static bool SupportsEventLogs
```
Comment (danmosemsft) [inline src/System.Diagnostics.EventLog/tests/Helpers.cs]: I would just write 
```c#
public static bool SupportsEventLogs { get => PlatformDetection.IsNotWindowsNanoServer && !PlatformDetection.IsWindows7 }
```

**Comment by danmosemsft** [comment]
In the change description (forming the topmost comment) please include the ID of the issue this relates to. Eg. "Relates to #12345". Don't use "Fixes" or similar words unless it is an actual fix, as that will cause github to close the issue. But you need a related link like this or nobody has any idea why you are doing it or did it.

**Comment by danmosemsft** [comment]
@Anipik I notice you're still using the same branch (newbr). I recommend you create a new branch for every PR.

**Comment by danmosemsft** [comment]
(a branch off master -- so checkout master before creating the new branch)

**Comment by Anipik** [comment]
@danmosemsft  I would first like sync my master with the dotnet/master then create a new branch for the pr ?

**Comment by danmosemsft** [comment]
System.Net.Security.Tests failures are unrelated active issue

**Pull request merged**